README.md

```diff
@@ -73,6 +74,7 @@ The Azure Key Vault Provider offers two modes for accessing a Key Vault instance
     | provider       | yes      | specify name of the provider                                    | ""            |
     | usePodIdentity | no       | specify access mode: service principal or pod identity          | "false"       |
     | keyvaultName   | yes      | name of a Key Vault instance                                    | ""            |
+    | cloudName      | no       | Name of the azure cloud based on azure go sdk                   | ""            |
```

Comment:
nit: can we add allowed values for cloud names -  `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `AzureGermanCloud`

Reply:
Done